src/routes/pokemonRoutes.ts

```diff
@@ -0,0 +1,23 @@
+import { Router } from "express";
+import {
+  getAllPokemons,
+  getOwnedPokemons,
+  getPokemonById,
+  getRandomPokemon,
+  searchPokemons,
+} from "../controllers/pokemonController";
+import {
+  validatePokemonId,
+  validateSearchPokemons,
+} from "../middleware/validationMiddleware";
+
+const router = Router();
+
+/*** GET ***/
+router.get("/pokemons", getAllPokemons);
+router.get("/pokemons/owned", getOwnedPokemons);
+router.get("/pokemons/:id", validatePokemonId, getPokemonById);
```

Comment:
you can add the pokemons in the index.ts - app.use("/api/pokemons", pokemonRoutes);